libs/auth/src/piv.ts

```diff
@@ -9,14 +9,6 @@ import { STATUS_WORD } from './apdu';
  * PIV spec.
  */
 
-/**
- * standard tags for PIV TLVs
- */
-export const GENERATE_RSA_PUBLIC_KEY_RESPONSE_TAG = Buffer.of(0x7f, 0x49); // eslint-disable-next-line vx/gts-jsdoc
-export const RSA_PUBLIC_KEY_MODULUS_TAG = 0x81; // eslint-disable-next-line vx/gts-jsdoc
-export const RSA_PUBLIC_KEY_EXPONENT_TAG = 0x82; // eslint-disable-next-line vx/gts-jsdoc
-export const SEQUENCE_TAG = 0x30;
```

Comment:
`GENERATE_RSA_PUBLIC_KEY_RESPONSE_TAG` already exists `GENERATE_ASYMMETRIC_KEY_PAIR.RESPONSE_TAG`.

`RSA_PUBLIC_KEY_MODULUS_TAG` and `RSA_PUBLIC_KEY_EXPONENT_TAG` can also be grouped under `GENERATE_ASYMMETRIC_KEY_PAIR`.

`SEQUENCE_TAG` isn't a PIV-specific tag (as far as I can tell) so doesn't make sense in this file.